dot/network/block_announce.go

```diff
@@ -198,7 +199,7 @@ func (s *Service) getBlockAnnounceHandshake() (Handshake, error) {
 	}, nil
 }
 
-func (s *Service) validateBlockAnnounceHandshake(peer peer.ID, hs Handshake) error {
+func (s *Service) validateBlockAnnounceHandshake(peer peer.ID, hs Handshake, stream libp2pnetwork.Stream) error {
```

Comment:
don't need to add `stream` in this function as that would require changing all the handler functions, it should be stored in the `createNotificationsMessageHandler` function

Reply:
changing this function signature would also require exposing streams to external packages such as grandpa which we don't want